tests/uvdata/test_mwa_corr_fits.py

```diff
@@ -1050,7 +1052,7 @@ def test_van_vleck_int():
     # read in file corrected using integrate.quad with 1e-10 precision
     uv2 = UVData()
     uv2.read(filelist[10])
-    assert np.allclose(uv1.data_array, uv2.data_array)
+    assert uv1._data_array == uv2._data_array
```

Comment:
Should we change this (and the other bare `assert ..==` calls below in this file) to use `np.testing.assert_allclose` too?

Reply:
So this actually does use an allclose call, I'm calling it via the UVParameter `_data_array`. But I think I did this early in my changes and I'm happy to change it to an `np.testing.assert_allclose` call on the actual arrays instead.

Reply:
I think in general the == comparison is good for users, but for our testing, if this ever fails, the numpy utility will help us figure out why.

Reply:
I'll make the change.

Reply:
Fixed. I also realized that I had a couple changes in files that I hadn't saved, so they didn't end up in earlier commits, so I added them too.